### ISSUE TITLE
Cannot find module ".prisma/client/index" from "/mnt/c/.../myrepo/node_modules/@prisma/client/index.js"

### ISSUE THREAD

**Issue by viveleroi** [oven-sh/bun#4847]
### What version of Bun is running?

1.0.0

### What platform is your computer?

Windows 11 (Microsoft Windows NT 10.0.22621.0 x64)

### What steps can reproduce the bug?

Use create-t3-app to create an app with prisma. 
Run the app with `bun run dev`

### What is the expected behavior?

No errors.

### What do you see instead?

```
bun dev
$ next dev
[4.26ms] ".env"
- info Loaded env from /mnt/c/Users/botsk/web-projects/jazzle/.env
[1.52ms] ".env"
- ready started server on localhost:3000, url: http://localhost:3000
- event compiled client and server successfully in 553 ms (18 modules)
- wait compiling...
- event compiled client and server successfully in 232 ms (18 modules)
[1.12ms] ".env"
[1.68ms] ".env"
- info Loaded env from /mnt/c/Users/botsk/web-projects/jazzle/.env
- info Loaded env from /mnt/c/Users/botsk/web-projects/jazzle/.env
- wait compiling / (client and server)...
Watchpack Error (watcher): Error: AccessDenied
Watchpack Error (watcher): Error: AccessDenied
Watchpack Error (watcher): Error: AccessDenied
Watchpack Error (watcher): Error: AccessDenied
Watchpack Error (watcher): Error: AccessDenied
- event compiled client and server successfully in 6.1s (332 modules)
- wait compiling...
Watchpack Error (watcher): Error: AccessDenied
Watchpack Error (watcher): Error: AccessDenied
Watchpack Error (watcher): Error: AccessDenied
- event compiled client and server successfully in 315 ms (332 modules)
- wait compiling /api/auth/[...nextauth] (client and server)...
- wait compiling /api/trpc/[trpc] (client and server)...
- event compiled successfully in 367 ms (95 modules)
- event compiled successfully (102 modules)
- error {
  name: "ResolveMessage",
  source: "server",
  message: "Cannot find module \".prisma/client/index\" from \"/mnt/c/Users/botsk/web-projects/jazzle/node_modules/@prisma/client/index.js\""
}
- error {
  name: "ResolveMessage",
  source: "server",
  message: "Cannot find module \".prisma/client/index\" from \"/mnt/c/Users/botsk/web-projects/jazzle/node_modules/@prisma/client/index.js\""
}
```

**Comment by BaileyNepe** [comment]
Running `bun run prisma generate` after running `bun install` fixed this for me

**Comment by viveleroi** [comment]
I had tried that and still got the errors, originally I was getting errors about the installed binaries for my OS which this did fix. I'll try again

**Comment by MegaR** [comment]
I think this is the same as https://github.com/oven-sh/bun/issues/4848#issuecomment-1713437567

**Comment by viveleroi** [comment]
That does make some of the console spam go away but the error is the same, except different:

```
- error Error: Cannot find module '.prisma/client/index'
Require stack:
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/@prisma/client/index.js
- /mnt/c/Users/botsk/web-projects/jazzle/.next/server/pages/api/auth/[...nextauth].js
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/next/dist/server/future/helpers/module-loader/node-module-loader.js
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/next/dist/server/future/helpers/module-loader/route-module-loader.js
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/next/dist/server/next-server.js
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/next/dist/server/dev/next-dev-server.js
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/next/dist/server/next.js
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/next/dist/server/lib/render-server.js
- /mnt/c/Users/botsk/web-projects/jazzle/node_modules/next/dist/compiled/jest-worker/processChild.js
```

**Comment by zach-betz-hln** [comment]
> Running `bun run prisma generate` after running `bun install` fixed this for me

Same here, thanks.

**Comment by MegaR** [comment]
> Running `bun run prisma generate` after running `bun install` fixed this for me

It only words if you also have node installed because one of prisma scripts has `#!/usr/bin/env ts-node` at the top.
But this is more of a prisma issue and not a bun issue.

**Comment by vinibretzke** [comment]
> Running `bun run prisma generate` after running `bun install` fixed this for me

I did this but still getting the error: 

error: Cannot find module ".prisma/client/index" from "/home/bretzke/vscode-projects/bun-quickstart/node_modules/@prisma/client/index.js"

Anyone else still having the issue?

**Comment by veloxy** [comment]
> > Running `bun run prisma generate` after running `bun install` fixed this for me
> 
> I did this but still getting the error:
> 
> error: Cannot find module ".prisma/client/index" from "/home/bretzke/vscode-projects/bun-quickstart/node_modules/@prisma/client/index.js"
> 
> Anyone else still having the issue?

I am, the generate command isn't doing anything:

```
root@35e86b5f03cb:/app# bunx prisma generate
[0.02ms] ".env"
> Downloading Prisma engines for Node-API for debian-openssl-3.0.x [====================] 98%
root@35e86b5f03cb:/app# ls -lh node_modules/.prisma/
total 0
```

It seems to just stop without any errors.

Related https://github.com/oven-sh/bun/issues/5320

**Comment by zach-betz-hln** [comment]
If it helps, for comparison, my `prisma/schema.prisma` file has this config. The main difference being the `engineType`

```plain
generator client {
  provider   = "prisma-client-js"
  engineType = "binary"
}
```

**Comment by paperclover** [comment]
prisma has a postinstall command we are not running on install. technically is a duplicate of #606, will be fixed by #5077

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway and @paperdave in #7132

This will be part of the Bun v1.0.17 release, which ships in a few hours from the time of writing.
